MyArrayList.java

```diff
@@ -0,0 +1,145 @@
+package com.company;
+
+public class MyArrayList<T> implements MyList<T> {
+
+    private T[] arr;
+    private int size;
+
+    public MyArrayList() {
+        size = 0;
+        arr = (T[]) new Object[size];
+    }
+
+    public MyArrayList(T[] arr) {
+        this.arr = arr;
+        size = arr.length;
+    }
+
+    @Override
+    public void add(T value) {
+        if (size < arr.length) {
+            arr[size] = value;
+            size++;
+        } else {
+            T[] newArr = (T[]) new Object[size + 1];

```

Comment:
Better increase size of array not by 1, because each nex add operation you will copy array, better something like  new Object[size * 1]

Reply:
> Better increase size of array not by 1, because each nex add operation you will copy array, better something like new Object[size * 1]

well, the lecturer just said that this is my implementation of the list and I can do it as I want.

Reply:
yes this is your code, but this is pull request to some code base and it should correspond to mate  quality standards, currently they are already low to pass, we can not make it even lower